tests/unit/test_opds_publication.py

```diff
@@ -265,7 +274,7 @@ def test_parseEditionToPublication_ReaderFlagTrue(self, testPubEls, mocker):
             mocker.call('rights', {'source': 'testSource', 'license': 'testLicense', 'rightsStatement': 'testStatement'})
         ])
 
-        pubMocks['addLink'].assert_called_with({'href': 'https://digital-research-books-beta.nypl.org/read/testID', 'type': 'testType', 'rel': 'http://opds-spec.org/acquisition/open-access', 'identifier': 'readable'})
+        pubMocks['addLink'].assert_called_with({'href': 'https://drb-qa.nypl.org/read/testID', 'type': 'testType', 'rel': 'http://opds-spec.org/acquisition/open-access', 'identifier': 'readable'})
```

Comment:
The environment variable used by this function seemed to be different when I ran the tests individually vs as a set, and then seemed to also behave different on my machine vs QA. I wondered if it was being set by TestFlaskAPI in some cases but cleared to an empty string in others. In any case, adding the helper setup and teardown seemed to normalize it to a "test" environment.